### PR TITLE
Add new columns to crosswalk table

### PR DESCRIPTION
**Pull request by karenmajewicz**
This adds a few columns to the crosswalk table to indicate what changed about the field. 

I have started working on additional documentation for the fields that are un-crosswalkable and will do that in another PR.

### CODE REVIEW

File: docs/aboutAardvark.md [hunk 1]
```diff
@@ -145,32 +145,33 @@ The original schema features several descriptive metadata fields that only accep
...
-| `dc_rights_s`           | `dct_accessRights_s`    |
-| `dc_format_s`           | `dct_format_s`          |
-| `layer_id_s`            | `gbl_wxsIdentifier_s`   |
-| `dct_references_s`      | `dct_references_s`      |
-| `layer_slug_s`          | `id`                    |
-| `dc_identifier_s`       | `dct_identifier_sm`     |
-| `layer_modified_dt`     | `gbl_mdModified_dt`     |
-| `geoblacklight_version` | `gbl_mdVersion_s`       |
-| `suppressed_b`          | `gbl_suppressed_b`      |
+| GBL 1.0                 | GBL Aardvark            |new namespace |single to multiple |completely different URI name|
```
Comment (rmseifried) [inline docs/aboutAardvark.md:151]:
```suggestion
| GBL 1.0                 | OGM Aardvark            |new namespace |single to multiple |completely different URI name|
```

Comment (rmseifried) [inline docs/aboutAardvark.md]:
```suggestion
| GBL 1.0                 | GBL Aardvark            | Same | New Namespace | Single to Multiple | Completely Different URI |
```

File: docs/aboutAardvark.md [hunk 2]
```diff
@@ -145,32 +145,33 @@ The original schema features several descriptive metadata fields that only accep
...
+|:------------------------|:------------------------|:-------------:|:-----------------:|:---------------------------:| 
+| `dc_title_s`            | `dct_title_s`           |X         |                   ||
+| `dc_description_s`      | `dct_description_sm`    |X         |X                  ||
```
Comment (rmseifried) [inline docs/aboutAardvark.md:154]:
```suggestion
| `dc_description_s`      | `dct_description_sm`    ||X         |X                  ||
```

File: docs/aboutAardvark.md [hunk 3]
```diff
@@ -145,32 +145,33 @@ The original schema features several descriptive metadata fields that only accep
...
+| `dc_language_sm`        | `dct_language_sm`       |X         |                   ||
```
Comment (rmseifried) [inline docs/aboutAardvark.md:155]:
```suggestion
| `dc_language_sm`        | `dct_language_sm`       ||X         |                   ||
```

**Review by rmseifried — CHANGES_REQUESTED** [review]
Suggest changing "GBL Aardvark" to "OGM Aardvark," and also adding a columns "Same" (or some other name) to indicate more clearly which fields can be crosswalk without changes.

**Comment by karenmajewicz** [comment]
I think I got all the changes in the latest commit. (some of them got overwritten in this interface.) One thing I was running into is that the width of the default layout is kind of narrow. I made some CSS changes that may or may not be problematic

**Pull request merged**